src/cobalt_model/pagination_config.rs

```diff
@@ -0,0 +1,108 @@
+use liquid;
+use std::borrow::Cow;
+use std::vec::Vec;
+use super::SortOrder;
+
+// struct Trails {
+//   before: i32,
+//   after: i32,
+// }
+
+#[derive(Clone, Debug)]
+pub(crate) struct PaginationCfg {
+  pub(crate) include: String,
```

Comment:
Should this be an `Option` for when pagination is disabled?

Or better, make the frontmatter tracks `PaginationCfg` in an `Option` so we move that to a `None` when this is disabled.

Reply:
You're right, I'm using `"None"` instead of `None` -_-